Client/Frontend/Menu/Items/MenuItemView.swift

```diff
@@ -0,0 +1,106 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+/**
+ * creates a view that consists of an image and a title.
+ * the image view is displayed at the center top of the
+ **/
+public class MenuItemView: UIControl {
+
+    var padding: CGFloat = 5.0
+
+    private(set) public var imageView: UIImageView
+    private(set) public var titleLabel: UILabel
+
+    private var previousLocation: CGPoint?
+    public init() {
+        imageView = UIImageView()
+        titleLabel = UILabel()
```

Comment:
Nit: Can move these alongside the property definitions above. You can also move the setup code for the label into a lazy block to clean up the init call.